grails-app/views/_common/modals/_deleteDialogCustomAction.gsp

```diff
@@ -26,9 +26,3 @@ This is the standard dialog that initiates the delete action.
         </div>
```

Comment:
Pls remove old and deprecated comments. Also since only _confDetails.gsp uses this, move it into the same package